lizmap/tooltip.py

```diff
@@ -379,6 +390,18 @@ def _generate_value_relation(widget_config: dict, name: str):
                             )
         return field_view
 
+    @staticmethod
+    def _generate_text_label(label: str, expression: str):
+        text = '''
+                    <p><b>{0}</b>
```

Comment:
```suggestion
                    <p><strong>{0}</strong>
```

According to @nboisteault `<b>` should be repaced by `<strong>`, CF https://www.w3schools.com/tags/tag_b.asp see **Tips and Notes**